app/views/SessionExpiredView.scala.html

```diff
@@ -33,7 +33,7 @@
 
 
     <div class="section">
-        <a id = "govuk-link" class="button" role = "button" href="@appConfig.loginUrl">
+        <a id = "govuk-link" class="button" role = "button" href="claimantType">
```

Comment:
Do we not need to put the route to claimantType here?

Reply:
the route to the claimantType cannot be added to the `navigator.scala `class as the timeout page is effective upon every page within the service.

Reply:
Don't need to add anything to he navigator, Simply add @routes.ClaimantTypeController.onPageLoad()?